tests/config.neon

```diff
@@ -21,7 +22,7 @@ parameters:
             new-api:
                 "2.0":
                     '/channels':
-                        post:
+                        post|put:
```

Comment:
Could you please provide some real-world use case. Because this looks to me like you want to use same `Action` to multiple things. Hence the `Action` has multiple responsibilities.